.github/workflows/build.yml

```diff
@@ -9,6 +9,9 @@ on:
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
 
+permissions:
+  contents: read
```

Comment:
This action also:
1. Writes comments on a PR
2. Uploads files to AWS

Reply:
Uploading the files is line 94 in this file. Commenting actually seems to be in a separate workflow `deploy` which probably needs permissions added too